docs/user/dashboard/dashboard.asciidoc

```diff
@@ -3,520 +3,286 @@
 
 [partintro]
 --
+**_Visualize your data with dashboards._**
 
-A _dashboard_ is a collection of panels that you use to analyze your data. On a dashboard, you can add a variety of panels that
-you can rearrange and tell a story about your data. Panels contain everything you need, including visualizations,
-interactive controls, markdown, and more.
-
-With *Dashboard*, you can:
-
-* Add multiple panels to see many aspects and views of your data in one place.
-
-* Arrange panels for analysis and comparison.
-
-* Add text and images to provide context to the panels and make them easy to consume.
-
-* Create and apply filters to focus on the data you want to display.
-
-* Control who can use your data, and share the dashboard with a small or large audience.
-
-* Generate reports based on your findings.
-
-To begin, open the main menu, click *Dashboard*, then click *Create dashboard*.
+The best way to understand your data is to visualize it. With dashboards, you can turn your data from one or more <<index-patterns, index patterns>> into a collection of panels 
```

Comment:
The tag line and the first sentence are very similar.  How about using  "The best way to understand your data is to visualization..." as your tag line?